src/many-ledger/src/json.rs

```diff
@@ -126,12 +126,21 @@ impl AccountJson {
     }
 }
 
+#[derive(serde::Deserialize, Clone, Debug)]
+pub struct SymbolMetaJson {
```

Comment:
Add memo please. It _should_ serialize using Serde (and if not, not a bad idea to add it).

Reply:
A `memo` for what exactly?

The `tokens.create` endpoint doesn't have a `memo` field.

Reply:
The `memo` for the "other calls" is logged in the event. The only token creation endpoint with this behavior is `tokens.update`.

The `memo` in `token-extended-info` is part of the extended info and can be retrieved using `tokens.info`. It is NOT part of the event. Only the extended info attribute indices are.

How would you like to handle this?